addons/hr_holidays/models/hr_holidays.py

```diff
@@ -372,6 +376,20 @@ def unlink(self):
     ####################################################
 
     @api.multi
+    def subscribe_followers(self):
```

Comment:
I don't know this method :p 
do I just need to add a method :
      @api.multi
      def _track_subtype(self, init_values):
         self.ensure_one()
         if 'state' in init_values and self.state == 'confirm':
             self.subscribe_followers()
         return super(AccountInvoice, self)._track_subtype(init_values)

Instead of putting it in create and write ?